lib/c-api/src/wasm_c_api/wasi/mod.rs

```diff
@@ -132,14 +133,31 @@ pub unsafe extern "C" fn wasi_config_mapdir(
     true
 }
 
+#[no_mangle]
+pub extern "C" fn wasi_config_capture_stdout(config: &mut wasi_config_t) {
+    config.inherit_stdout = false;
+}
+
 #[no_mangle]
 pub extern "C" fn wasi_config_inherit_stdout(config: &mut wasi_config_t) {
     config.inherit_stdout = true;
 }
+
+#[no_mangle]
+pub extern "C" fn wasi_config_capture_stderr(config: &mut wasi_config_t) {
+    config.inherit_stderr = false;
+}
+
 #[no_mangle]
 pub extern "C" fn wasi_config_inherit_stderr(config: &mut wasi_config_t) {
     config.inherit_stderr = true;
 }
+
+#[no_mangle]
+pub extern "C" fn wasi_config_capture_stdin(config: &mut wasi_config_t) {
+    config.inherit_stdin = false;
+}
+
```

Comment:
What does it mean to capture stdin here?

I don't think we have any APIs to make use of this, perhaps keep it commented out so we don't export this symbol until we have a use for it.

Reply:
Let's keep `wasi_config_capture_stdin` apart for the moment :-).